ui/components/app/create-new-vault/create-new-vault.scss

```diff
@@ -0,0 +1,53 @@
+.create-new-vault {
+  &__form {
+    display: flex;
+    flex-direction: column;
+    width: 360px;
+  }
+
+  &__srp-shown {
+    @include Paragraph;
+
+    padding: 8px 16px;
+  }
+
+  &__srp-section {
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__srp-label {
+    margin-bottom: 8px;
+  }
+
+  &__srp-error {
+    margin-top: 4px;
+  }
+
+  &__show-srp {
+    margin-top: 16px;
+    margin-bottom: 16px;
+  }
+
+  &__show-srp-label {
+    margin-left: 8px;
+  }
+
+  &__terms {
+    margin-top: 16px;
+    margin-bottom: 16px;
+  }
+
+  &__terms-label {
+    margin-left: 8px;
+  }
+
+  &__terms-link {
+    color: var(--primary-1);
+  }
+
+  &__submit-button#{&}__submit-button {
```

Comment:
The `#{...}` is for [Interpolation](https://sass-lang.com/documentation/interpolation), which is needed when repeating an ampersand in a classname. The class is doubled up here so that it has a higher specificity than the button classes it's meant to override. For some reason those are higher.

Reply:
Ah, interesting. So is the final class name here `.create-new-vault__submit-button .create-new-vault__submit-button`?

Reply:
Yep, exactly

Reply:
I think this is because the UI styles are imported _after_ the app styles here: https://github.com/MetaMask/metamask-extension/blob/develop/ui/css/index.scss#L12

The UI components are our "base" components, so we should be able to override the styles in our app components and pages. So this ordering seems totally wrong. But changing this would impact all of our styles, so it could result in subtle style breakages all over the app, which has me hesitant to try fixing it.

Reply:
Yeah, I think I've run across that before too. Alas 😞